composer.json

```diff
@@ -15,14 +15,14 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
-        "symfony/dependency-injection": "~3.0|~4.0|~5.0",
-        "symfony/config": "~3.0|~4.0|~5.0",
-        "symfony/yaml": "~3.0|~4.0|~5.0",
-        "symfony/finder": "~3.0|~4.0|~5.0",
-        "symfony/filesystem": "~3.0|~4.0|~5.0",
-        "symfony/event-dispatcher": "~3.0|~4.0|~5.0",
+        "php": "^7.4|^8.0|^8.1",
```

Comment:
This is not necessary as `^8.0` already includes 8.1:
https://getcomposer.org/doc/articles/versions.md#caret-version-range-